src/app/components/account/account-name.tsx

```diff
@@ -6,5 +6,7 @@ interface AccountNameLayoutProps {
   children: React.ReactNode;
 }
 export const AccountNameLayout = memo(({ children }: AccountNameLayoutProps) => (
-  <styled.p textStyle="label.01">{children}</styled.p>
+  <styled.span fontWeight={500} textStyle="label.02">
```

Comment:
This is the same code as in `account-total-balance`. We should refactor this to share